cluster-scope/base/core/namespaces/octo-et-security-demo-next/namespace.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/display-name: "OCTO ET Security: Demo for NEXT"
+    openshift.io/requester: octo-et-security
+    op1st/project-owner: octo-et-security
+    op1st/onboarding-issue: "https://github.com/operate-first/support/issues/628"
+    op1st/docs: "https://github.com/orgs/redhat-et/teams/security"
```

Comment:
This is internal documentation (anybody not on the team cannot access this), but the purpose of this is to have external facing documentation. Not the biggest deal but something worth bringing up.

Reply:
I think this is meant to be the docs link for that project. And this is the best I could come up with. It'll be used e.g. for the workload monitoring at https://grafana.operate-first.cloud/d/opf-overview/workload-overview?orgId=1